### PR TITLE
Fix eeprom vendor extension TLV number counting issue

### PR DESCRIPTION
**Pull request by keboliu**
In TlvInfoDecoder class using a member variable _TLV_NUM_VENDOR_EXT to counting the TLV numbers in the system EEPROM, each time update_eeprom_db be called, _TLV_NUM_VENDOR_EXT will be accumulated, which will cause EEPROM extension TLV number counting issue. 

The fix is to replace this member variable with a local variable in the function update_eeprom_db to eliminate the TLV number counting issue.

add extra blank lines to align with PEP8

### CODE REVIEW

File: sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py [hunk 1]
```diff
@@ -383,8 +393,8 @@ def update_eeprom_db(self, e):
             else:
                 tlv_index += ord(e[tlv_index + 1]) + 2
 
-        if self._TLV_NUM_VENDOR_EXT:
-            fvs['Num_vendor_ext'] = self._TLV_NUM_VENDOR_EXT
+        if vendor_ext_tlv_num:
```
Comment (nazariig) [inline sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py]: @keboliu this is not obvious: it would be better to have:
`if vendor_ext_tlv_num > 0:`

File: sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py [hunk 2]
```diff
@@ -383,8 +393,8 @@ def update_eeprom_db(self, e):
...
+            fvs['Num_vendor_ext'] = vendor_ext_tlv_num
```
Comment (nazariig) [inline sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py]: @keboliu convert to a string?

File: sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py [hunk 3]
```diff
@@ -662,11 +680,14 @@ def encoder(self, I, v):
 
         return chr(I[0]) + chr(len(value)) + value
 
+
```
Comment (nazariig) [inline sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py]: @keboliu please keep in mind: formatting improvements should go in a separate PR.

File: sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py (hunk 1 above)
Comment (keboliu) [reply sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py]: revised.

File: sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py (hunk 2 above)
Comment (keboliu) [reply sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py]: revised

File: sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py (hunk 3 above)
Comment (keboliu) [reply sonic_platform_base/sonic_eeprom/eeprom_tlvinfo.py]: do the formatting improvements when touching a file could make the work more easy to move forward step by step, no harm I guess.
